auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -388,11 +389,11 @@ public static int getDefaultTheme() {
             }
 
             String type = ProviderUtils.providerIdToAccountType(userInfo.getProviderId());
-
-            credentials.add(new Credential.Builder(
-                    TextUtils.isEmpty(user.getEmail()) ? user.getPhoneNumber() : user.getEmail())
-                    .setAccountType(type)
-                    .build());
+            if (type == null) {
```

Comment:
Let's just add a comment here for future-us explaining why we're doing this hack with the "pass" credential.

Reply:
Yeah just explaining why the password doesn't matter (since we're just creating a credential for deletion)

Reply:
Sweet, will do!